docs/_docs/components/bar.md

```diff
@@ -59,8 +59,8 @@ The bar component has the following structure:
     </BarStart>
     <BarEnd>
       <BarItem>
-        <Button Color="Color.Primary">Sign up</Button>
-        <Button Color="Color.Secondary">Log in</Button>
+        <button Color="Color.Primary">Sign up</button>
```

Comment:
Why `button` instead of `Button`? Is this a mistake?

Reply:
Oops! Yes this is a mistake.
Must've auto save formatting